app/common/elasticsearchclient.py

```diff
@@ -2820,24 +2733,54 @@ def _get_aggregation_on_rna_expression_tissue(filters, filters_func, size,
                     "data": {
```

Comment:
why no `addict` here?

Reply:
@elipapa lol, @apierleoni got mad at me and I stopped using addict everywhere